flowzone.yml

```diff
@@ -1154,35 +1220,156 @@ jobs:
           echo "sha=$(echo $response | jq -r .sha)" >> "${GITHUB_OUTPUT}"
           echo "json=$(echo $response | jq -c .)" >> "${GITHUB_OUTPUT}"
 
-      # https://docs.github.com/en/rest/git/refs?apiVersion=2022-11-28#update-a-reference
-      - name: Update branch reference
-        if: github.event.pull_request.merged == true && steps.create_commit.outputs.sha != ''
+      - *updateGitReference
+      - *createGitReference
+
+      - <<: *getGitHubAppToken
+        id: pull_requests_write
+        if: inputs.disable_versioning != true && inputs.release_notes == true
+        with:
+          <<: *getGitHubAppTokenWith
+          permissions: >-
+            {
+              "contents": "read",
+              "metadata": "read",
+              "pull_requests": "write"
+            }
+
+      # Get Renovate release notes from PR body, massage a little and draft a comment.
+      - name: Draft PR comment
```

Comment:
Could we put the release notes, PR comment, and Zulip message in another job that depends on versioned_source? It will make it easier to track the errors and we don't have to remember to put `inputs.release_notes == true` on every step if the new job requires it.